package.json

```diff
@@ -316,13 +322,16 @@
     "lodash": "4.17.21",
     "lodash-move": "1.1.1",
     "lodash-webpack-plugin": "0.11.5",
+    "lodash.sortby": "^4.7.0",
     "mem": "4.0.0",
     "mini-css-extract-plugin": "0.9.0",
     "moment": "2.24.0",
+    "object-assign": "^4.1.1",
     "pofile": "1.0.10",
     "postcss-flexbugs-fixes": "4.2.1",
     "postcss-loader": "3.0.0",
     "postcss-overrides": "3.1.4",
+    "prepend-http": "^3.0.1",
```

Comment:
It seems that this is the cause of the failing test. I would use the same that is right now in the lock.

Reply:
```
 WARN  deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
 WARN  deprecated babel-plugin-react-intl@5.1.17: this package has been renamed to babel-plugin-formatjs
 WARN  deprecated draft-js-buttons@2.0.2: use @draft-js-plugins/buttons >=v4 instead
 WARN  deprecated draft-js-plugins-editor@2.1.1: use @draft-js-plugins/editor >=v4 instead
 WARN  deprecated draft-js-inline-toolbar-plugin@2.0.3: use @draft-js-plugins/inline-toolbar >=v4 instead
```